src/emailpro/account/add/emailpro-account-add.controller.js

```diff
@@ -161,13 +161,21 @@ angular.module('Module.emailpro.controllers')
   });
 
 angular.module('Module.emailpro.controllers')
-  .controller('EmailProOrderAccountCtrl', ($scope, $stateParams, EmailPro) => {
+  .controller('EmailProOrderAccountCtrl', ($scope, $stateParams, EmailPro, User) => {
     // default values
     $scope.accountsToAdd = {
       duration: '12',
       accountsNumber: 1,
     };
 
+    User.getUser()
+      .then((currentUser) => {
```

Comment:
you could use destructive pattern `.then(({ ovhSubsidiary }))`

Reply:
This is done. Thanks :)